packages/eslint-plugin-edu-react/src/index.ts

```diff
@@ -1,13 +1,22 @@
-module.exports = {
+import useEffectDepsPresence from './rules/use-effect-deps-presence';
+
+const rules = {
+  'use-effect-deps-presence': useEffectDepsPresence,
+};
+
+const recommendedConfig = {
   extends: ['plugin:jsx-a11y/recommended'],
-  plugins: ['jsx-a11y', 'react', 'react-hooks'],
+  plugins: ['@chanzuckerberg/edu-react', 'jsx-a11y', 'react', 'react-hooks'],
   settings: {
     react: {
       version: 'detect',
     },
   },
   rules: {
-    'jsx-a11y/anchor-ambiguous-text': 'error',
+    // prettier-ignore
+    '@chanzuckerberg/edu-react/use-effect-deps-presence': 'error',
```

Comment:
Weirdly the "full" name (`eslint-plugin-edu-react` vs `edu-react`) doesn't work. No idea why (docs indicate it should).